index.html

```diff
@@ -166,51 +166,34 @@ <h4>Skills</h4>
               <li class="li--section">
                 <button class="btn--frame" type="button">Terminal</button>
               </li>
-
             </ul>
           </div>
         </li>
-
       </ul>
     </section>  
     <section class="contact--section" id="contact">
       <div class="contact--intro">
         <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
       </div>
-      <form class="form--section" action="https://formspree.io/f/xbjwrqlv" method="post">
-        <div class="contact--ul--section">
+
+      <form id="form" class="form--section" action="https://formspree.io/f/xbjwrqlv" method="post">
+        <div class="contact--ul--section desk--contact--section">
```

Comment:
- [ ] Currently, your form does not submit when I entered the correct details `lowercase characters in the email field` and entered my name and message. Please try fixing it so that your app works better. Or maybe there is something I'm not doing well then try displaying an error message describing my errors and what to do to fix it so that I can be able to send the form successfully. 💯